README.md

```diff
@@ -2,16 +2,20 @@
 
 Source code repo for the pelux.io website
 
-## How to build
+## How to build the web site locally
 
-This website uses [jekyll](https://jekyllrb.com/) to build static HTML. Those are the steps to build it:
+This website uses [jekyll](https://jekyllrb.com/) to build static HTML. You'll need some dependencies to build the site locally. These dependencies are best to pull from upstream since they rely on versions that are known to work together. Install the gems jekyll, bundler, and minima this way;
+
+    sudo gem install jekyll bundler minima
```

Comment:
While you were on vaccation I rewrote this part and tested it on a couple of different distributions with the new instructions, I also added a Jenkins file which utilizes the behaviour described in the new README. Perhaps you could rebase your changes on top of it (you need to do it anyway because we won't be able to merge it otherwise) and merge your changes to the new state of the README?